marshmallow/compat.py

```diff
@@ -45,6 +49,13 @@ def get_func_args(func):
     zip_longest = itertools.zip_longest
     from collections import OrderedDict
     OrderedDict = OrderedDict
+    try:
+        from collections import Mapping, Iterable, MutableSet
+    except ImportError:
+        from collections.abc import Mapping, Iterable, MutableSet
+    Mapping = Mapping
```

Comment:
I'm fine getting rid of this convention. I think it was done in order to avoid flake8 errors, but we are now ignoring this whole file in the flake8 config, so no use in keeping it.

Reply:
Ah, ok, that's why! Thank you @sloria for the explanation, for doing the remaining changes, and for merging them in :)